requirements/requirements.txt

```diff
@@ -1,8 +1,8 @@
 # needed to build gevent from source
-cython==0.23.4
+cython==0.27.3
 restkit==4.2.2
 iso8601==0.1.11
-jsonobject==0.7.1
+git+git://github.com/dimagi/jsonobject.git@8af0bf92761c3780492954366c068f3c598a7e8c
```

Comment:
Won't this get updated on pypi once the jsonobject PR is merged?

Reply:
I figured it should be tested here before 
It's released on pypi. May also be good for cory to profile this separately from his other work

Reply:
fine by me to have this merged independently whenever it's ready